model/post.go

```diff
@@ -13,32 +13,33 @@ import (
 )
 
 const (
-	POST_SYSTEM_MESSAGE_PREFIX = "system_"
-	POST_DEFAULT               = ""
-	POST_SLACK_ATTACHMENT      = "slack_attachment"
-	POST_SYSTEM_GENERIC        = "system_generic"
-	POST_JOIN_LEAVE            = "system_join_leave" // Deprecated, use POST_JOIN_CHANNEL or POST_LEAVE_CHANNEL instead
-	POST_JOIN_CHANNEL          = "system_join_channel"
-	POST_LEAVE_CHANNEL         = "system_leave_channel"
-	POST_JOIN_TEAM             = "system_join_team"
-	POST_LEAVE_TEAM            = "system_leave_team"
-	POST_ADD_REMOVE            = "system_add_remove" // Deprecated, use POST_ADD_TO_CHANNEL or POST_REMOVE_FROM_CHANNEL instead
-	POST_ADD_TO_CHANNEL        = "system_add_to_channel"
-	POST_ADD_TO_TEAM           = "system_add_to_team"
-	POST_REMOVE_FROM_CHANNEL   = "system_remove_from_channel"
-	POST_HEADER_CHANGE         = "system_header_change"
-	POST_DISPLAYNAME_CHANGE    = "system_displayname_change"
-	POST_PURPOSE_CHANGE        = "system_purpose_change"
-	POST_CHANNEL_DELETED       = "system_channel_deleted"
-	POST_EPHEMERAL             = "system_ephemeral"
-	POST_FILEIDS_MAX_RUNES     = 150
-	POST_FILENAMES_MAX_RUNES   = 4000
-	POST_HASHTAGS_MAX_RUNES    = 1000
-	POST_MESSAGE_MAX_RUNES     = 4000
-	POST_PROPS_MAX_RUNES       = 8000
-	POST_PROPS_MAX_USER_RUNES  = POST_PROPS_MAX_RUNES - 400 // Leave some room for system / pre-save modifications
-	POST_CUSTOM_TYPE_PREFIX    = "custom_"
-	PROPS_ADD_CHANNEL_MEMBER   = "add_channel_member"
+	POST_SYSTEM_MESSAGE_PREFIX  = "system_"
+	POST_DEFAULT                = ""
+	POST_SLACK_ATTACHMENT       = "slack_attachment"
+	POST_SYSTEM_GENERIC         = "system_generic"
+	POST_JOIN_LEAVE             = "system_join_leave" // Deprecated, use POST_JOIN_CHANNEL or POST_LEAVE_CHANNEL instead
+	POST_JOIN_CHANNEL           = "system_join_channel"
+	POST_LEAVE_CHANNEL          = "system_leave_channel"
+	POST_JOIN_TEAM              = "system_join_team"
+	POST_LEAVE_TEAM             = "system_leave_team"
+	POST_ADD_REMOVE             = "system_add_remove" // Deprecated, use POST_ADD_TO_CHANNEL or POST_REMOVE_FROM_CHANNEL instead
+	POST_ADD_TO_CHANNEL         = "system_add_to_channel"
+	POST_ADD_TO_TEAM            = "system_add_to_team"
+	POST_REMOVE_FROM_CHANNEL    = "system_remove_from_channel"
+	POST_HEADER_CHANGE          = "system_header_change"
+	POST_DISPLAYNAME_CHANGE     = "system_displayname_change"
+	POST_PURPOSE_CHANGE         = "system_purpose_change"
+	POST_CHANNEL_DELETED        = "system_channel_deleted"
+	POST_EPHEMERAL              = "system_ephemeral"
+	POST_CHANGE_CHANNEL_PRIVACY = "system_change_chan_privacy"
```

Comment:
Propose we'd use `system_change_channel_privacy` so it's more obvious, compared to `chan`.

Reply:
@jasonblais The `system_change_channel_privacy` will throw following error: `SqlPostStore.Save: store.sql_post.save.app_error, id=ob1n8fy8rb8upczub86eomtobr, Error 1406: Data too long for column 'Type' at row 1` cause field `Type` [has a 26 chars length](/mattermost/mattermost-server/blob/v4.5.0/store/sqlstore/post_store.go#L58) as maximum.

Reply:
Gotcha, this should be fine then.